src/util/readpool/mod.rs

```diff
@@ -0,0 +1,155 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+mod config;
+mod context;
+mod priority;
+
+use std::time::Duration;
+use futures::Future;
+use futures_cpupool as cpupool;
+
+use util;
+use util::futurepool::FuturePool;
+
+pub use self::config::Config;
+pub use self::context::Context;
+pub use self::priority::Priority;
+
+const TICK_INTERVAL_SEC: u64 = 1;
+
+pub struct ReadPool {
```

Comment:
I think `ReadPool` and its `Config` should be put in `util`. It's only for a specified business. What do you think? @zhangjinpeng1987